tests/system/action/committee/test_import_meeting.py

```diff
@@ -0,0 +1,534 @@
+                        {"id": 1, "meeting_id": 1, "name": "blup", "first_state_id": 1}
+                    ],
+                    "motion_state": [
+                        {
+                            "id": 1,
+                            "css_class": "line",
+                            "meeting_id": 1,
+                            "workflow_id": 1,
+                            "name": "test",
+                        }
+                    ],
+                    "projector": [{"id": 1, "meeting_id": 1}],
+                    "personal_note": [
+                        {"id": 1, "meeting_id": 1, "content_object_id": "motion/1"}
+                    ],
+                    "motion": [
+                        {
+                            "id": 1,
+                            "meeting_id": 1,
+                            "list_of_speakers_id": 1,
+                            "state_id": 1,
+                            "title": "bla",
+                        }
+                    ],
+                    "list_of_speakers": [
+                        {"id": 1, "meeting_id": 1, "content_object_id": "motion/1"}
+                    ],
+                    "tag": [
+                        {
+                            "id": 1,
+                            "meeting_id": 1,
+                            "tagged_ids": ["motion/1"],
+                            "name": "testag",
+                        }
+                    ],
+                },
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "meeting/2",
+            {
+                "name": "Test",
+                "description": "blablabla",
+                "committee_id": 1,
+                "enable_anonymous": False,
+            },
+        )
+        meeting_2 = self.get_model("meeting/2")
+        assert start <= meeting_2.get("imported_at", 0) <= start + 300
+        self.assert_model_exists(
+            "user/2", {"username": "test", "group_$2_ids": [1], "group_$_ids": ["2"]}
+        )
+        user_2 = self.get_model("user/2")
+        assert user_2.get("password", "")
+        self.assert_model_exists("projector/1", {"meeting_id": 2})
+        self.assert_model_exists("group/1", {"user_ids": [2]})
+        self.assert_model_exists("personal_note/1", {"content_object_id": "motion/2"})
+        self.assert_model_exists(
+            "tag/1", {"tagged_ids": ["motion/2"], "name": "testag"}
+        )
+        self.assert_model_exists("committee/1", {"meeting_ids": [2]})
+
+    def test_check_usernames(self) -> None:
+        self.set_models(
+            {
+                "committee/1": {},
+                "meeting/1": {},
+                "motion/1": {},
+                "user/1": {"username": "admin"},
+            }
+        )
+        response = self.request(
+            "committee.import_meeting",
+            {
+                "id": 1,
+                "meeting_json": {
```

Comment:
Can you define this once and reuse it in all tests?

Reply:
I def a test_models and use it in some tests.